plugins/importexport/native/filter/SubmissionNativeXmlFilter.inc.php

```diff
@@ -255,8 +256,10 @@ function addRepresentations($doc, $submissionNode, $submission) {
 		$representations = $representationDao->getBySubmissionId($submission->getId());
 		while ($representation = $representations->next()) {
 			$representationDoc = $exportFilter->execute($representation);
-			$clone = $doc->importNode($representationDoc->documentElement, true);
-			$submissionNode->appendChild($clone);
+			if ($representationDoc) {
+                $clone = $doc->importNode($representationDoc->documentElement, true);
```

Comment:
There's probably some missing error handling here that's not your doing -- but I'd rather throw a PHP error than have it continue processing with missing information. I'll leave this one out.